docs/Install-with-Docker.md

```diff
@@ -33,11 +33,19 @@ Using Docker will have the same benefits as using virtual machines, but with muc
 
 ![Docker containers vs Virtual machines](https://user-images.githubusercontent.com/75517/51078179-d4fec680-16b1-11e9-8ce6-87b5053f55dd.png)
 
-## Setup
+## PART 1 - Setup
 
 ### Software requirements
 
-The only requirements are [git](https://git-scm.com/download/) and Docker.
+The only requirements are [git](https://git-scm.com/download/) and Docker (https://docs.docker.com/docker-for-windows/install/).
+
+Be aware that Docker itself has several software requirements such as;
+- Windows 10 64-bit: Pro, Enterprise, or Education (Build 16299 or later).
```

Comment:
Docker also runs on macOS and GNU/Linux, so we need to specify that these requirements are for Windows specifically (and maybe move them below)

Reply:
It's better to have more information than less. I tried to install via windows and it just said "get docker". Little did I know getting docker would take 3-4 hours and require lots of steps. 

The guide before had no links, so the guide having a windows link is an improvement, if someone else who uses MAC or Linux wants to add a linux link they can but I don't have them.